platform-includes/logs/setup/unreal.mdx

```diff
@@ -0,0 +1,145 @@
+To enable logging in your Unreal Engine project, you need to configure the Sentry SDK with structured logging enabled.
+
+### Project Settings Configuration
+
+1. Open your project settings: **Project Settings > Plugins > Sentry**
+2. Check the **Enable Structured Logging** option
+
+### Programmatic Configuration
+
+Alternatively, you can enable logging programmatically when initializing the SDK:
+
+```cpp
+#include "SentrySubsystem.h"
+
+void ConfigureSentryWithLogs()
+{
+    USentrySubsystem* SentrySubsystem = GEngine->GetEngineSubsystem<USentrySubsystem>();
+
+    // Create settings with logging enabled
+    SentrySubsystem->InitializeWithSettings(FConfigureSettingsNativeDelegate::CreateLambda([=](USentrySettings* Settings)
+    {
+        Settings->EnableStructuredLogging = true;
+    }));
+}
+```
+
+### Advanced Configuration Options
+
+#### Automatic Unreal Engine Log Forwarding
+
+You can configure the SDK to automatically forward Unreal Engine's native `UE_LOG` calls to Sentry based on the enabled severity levels:
+
+```cpp
+USentrySubsystem* SentrySubsystem = GEngine->GetEngineSubsystem<USentrySubsystem>();
+
+// Configure automatic log forwarding programmatically
+SentrySubsystem->InitializeWithSettings(FConfigureSettingsNativeDelegate::CreateLambda([=](USentrySettings* Settings)
+{
+    Settings->EnableStructuredLogging = true;
+
+    // Enable specific severity levels for UE_LOG forwarding
+    Settings->EnableDebugLogs = false;
+    Settings->EnableInfoLogs = true;
+    Settings->EnableWarningLogs = true;
+    Settings->EnableErrorLogs = true;
+    Settings->EnableFatalLogs = true;
+
+    Settings->bSendBreadcrumbsWithStructuredLogging = false; // Send as structured logs instead of breadcrumbs
+}));
+```
+
+#### Log Category Filtering
+
+You can filter which log categories are sent to Sentry:
+
+```cpp
+// Configure category filtering
+USentrySubsystem* SentrySubsystem = GEngine->GetEngineSubsystem<USentrySubsystem>();
+
+// Create settings with logging enabled
+SentrySubsystem->InitializeWithSettings(FConfigureSettingsNativeDelegate::CreateLambda([=](USentrySettings* Settings)
+{
+    Settings->EnableStructuredLogging = true;
+    
+    // Only forward logs from specific categories
+    TArray<FString> AllowedCategories;
+    AllowedCategories.Add(TEXT("LogGameFlow"));
+    AllowedCategories.Add(TEXT("LogPlayerSystem"));
+    AllowedCategories.Add(TEXT("LogSentrySdk"));
```

Comment:
Maybe we don't add this to the snippet and end up with users copy-pasting this into their game. 😁